### PR TITLE
added React Native with Expo modules on Workflow options

### PR DESCRIPTION
**Pull request by efstathiosntonas**
this option/variant was missing from `bug-report.yml`

### CODE REVIEW

**Comment by efstathiosntonas** [comment]
I don't know if the already existent workflows trigger any CI when the issue was submitted or it's just for reference 🤔

**Comment by efstathiosntonas** [comment]
sorry for the commit mess, please squash it :/

**Comment by tjzel** [comment]
Don't worry about the commits, we always auto-squash 😄

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 1]
```diff
@@ -107,6 +107,7 @@ body:
       description: How your application is managed? Not sure? Read [this part](https://docs.expo.dev/introduction/managed-vs-bare/) of Expo documentation
       options:
         - React Native (without Expo)
+        - React Native with Expo modules
         - Expo bare workflow
         - Expo managed workflow
```
Comment (tjzel) [inline .github/ISSUE_TEMPLATE/bug-report.yml:112]: According to @lukmccall it's outdated anyway. We can make it more actual:

```suggestion
      description: How your application is managed?
      options:
        - React Native 
        - Expo Go
        - Expo Dev Client
```

**Review by tjzel — CHANGES_REQUESTED** [review]

**Comment by efstathiosntonas** [comment]
@tjzel you're right!

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 2]
```diff
@@ -104,11 +104,11 @@ body:
     id: workflow
     attributes:
       label: Workflow
-      description: How your application is managed? Not sure? Read [this part](https://docs.expo.dev/introduction/managed-vs-bare/) of Expo documentation
+      description: How your application is managed?
```
Comment (tjzel) [inline .github/ISSUE_TEMPLATE/bug-report.yml:107]: When I read this again it sounded weird 😅 

```suggestion
      description: How is your application managed?
```

**Review by tjzel — APPROVED** [review]

**Pull request merged**